### PR TITLE
Some cleanup within OC_Image

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: lib/private/legacy/image.php [hunk 1]
```diff
@@ -83,11 +83,6 @@ public function __construct($imageRef = null, $logger = null) {
 			$this->logger = \OC::$server->getLogger();
 		}
 
-		if (!extension_loaded('gd') || !function_exists('gd_info')) {
```
Comment (PVince81) [inline lib/private/legacy/image.php]: check not needed any more ?

Comment (PVince81) [inline lib/private/legacy/image.php]: even if needed, should like happen in setup checks anyway.

**Comment by PVince81** [comment]
See comment, otherwise :+1:

File: lib/private/legacy/image.php (hunk 1 above)
Comment (DeepDiver1975) [inline lib/private/legacy/image.php]: a mandatory setup requirement - https://github.com/owncloud/core/blob/master/lib/private/legacy/util.php#L748

**Pull request merged**